.prettierrc

```diff
@@ -0,0 +1,9 @@
+{
+  "semi": true,
+  "tabWidth": 2,
+  "printWidth": 100,
+  "singleQuote": true,
+  "trailingComma": "none",
+  "jsxBracketSameLine": true
```

Comment:
```suggestion
  "bracketSameLine": true
```

jsxBracketSameLine  is deprecated https://prettier.io/docs/en/options.html#deprecated-jsx-brackets

Reply:
How do you know that jsxBracketSameLine is deprecated? By experience? (Just surprised how you can know the correct usage of the packages.. Just as you found out that the firebase compat libraries are not suggested to be used.)